workflows/VGP-assembly-v2/Mitogenome-assembly-VGP0/Mitogenome-Assembly-VGP0.ga

```diff
@@ -15,7 +15,7 @@
     ],
     "format-version": "0.1",
     "license": "CC-BY-4.0",
-    "name": "Mitogenome-Assembly-VGP0",
+    "name": "Mitogenome Assembly VGP0",
```

Comment:
"name": "Mitogenome Assembly with Hifi data VGP0",

Reply:
is it maybe time to remove the VGPx suffix from all names?